.circleci/config.yml

```diff
@@ -26,7 +26,28 @@ jobs:
                        -scheme 'Automattic-Tracks-OSX' \
                        -destination 'platform=OS X,arch=x86_64' \
                        test  | tee logs/build-and-test.log | xcpretty -r junit
+  CheckVersionConsistency:
+    docker:
+      - image: cimg/base:2020.01
+    steps:
+      - checkout
+      - run:
+          name: Check pod version consistency
+          command: |
+            SRC_FILE=Automattic-Tracks-iOS/TracksConstants.m
+            PODSPEC_FILE=Automattic-Tracks-iOS.podspec
 
+            SOURCE_VERS=$(sed -n s/'^.* TracksLibraryVersion = @"\(.*\)";.*$'/'\1'/p $SRC_FILE)
+            POD_VERS=$(sed -n s/'^ *s.version *= \([^ ]*\).*$'/'\1'/p $PODSPEC_FILE | tr -d \'\")
```

Comment:
Note: It's easiest to have `sed` match the version _with_ the quotes (be it simple or double quotes) then have `tr` remove the quotes, than trying to use `["']` in the regex to match any kind of quotes.

This is because escaping `["']` within the single-quote that wraps the whole regex would make it both tricky to read and creates quite a mess in escape sequences required.